tests/e2e/pact/resources/common.ts

```diff
@@ -0,0 +1,27 @@
+import * as fhir from "../models/fhir/fhir-resources"
+import {JestPactOptions} from "jest-pact"
+import path from "path"
+import {ExampleFile} from "../models/files/example-file"
+
+export const pactOptions: JestPactOptions = {
+  spec: 3,
+  consumer: `nhsd-apim-eps-test-client+${process.env.PACT_VERSION}`,
+  provider: `nhsd-apim-eps+prepare+${process.env.PACT_VERSION}`,
```

Comment:
provider will need parametising here, it's nhsd-apim-eps{-sandbox}+{prepare|convert|process}+${process.env.PACT_VERSION}